tests/gce_disk_integration_test.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests for GCE scratch disks."""
+
+import os
+import unittest
+
+from perfkitbenchmarker import pkb
+from perfkitbenchmarker import test_util
+
+MOUNT_POINT = '/scratch'
+
+
+@unittest.skipUnless('PERFKIT_INTEGRATION' in os.environ,
+                     'PERFKIT_INTEGRATION not in environment')
+class GcpScratchDiskIntegrationTest(unittest.TestCase):
```

Comment:
After talking with Skyler in person, I added a section to the README discussing the integration tests, and put small docstrings in each of the integration tests classes pointing to the README. This lets me avoid repeating the same information in the docstring for each integration test, but still keeps it visible and available to users.